spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -885,20 +885,11 @@ point in time of the transaction.
 Because read-only transactions are non-locking and can be performed on points in time in the past, these
 are recommended for functions that do not perform write operations.
 
-===== Declarative transaction management
+===== @Transactional Annotation
 
-To use `@Transactional` in [declarative spring transaction management](https://docs.spring.io/spring/docs/current/spring-framework-reference/data-access.html#transaction-declarative)
-`SpannerTransactionManager` has to be registered in Spring.
-
-Example:
-```
-@Bean
-public SpannerTransactionManager transactionManager() {
```

Comment:
I don't think we necessarily need to remove this from documentation. Some people may not be using the starter. In any case, if you do remove it, at least we need to clarify that the auto-config/starter will create the `SpannerTransactionManager` needed to enable it.

Reply:
Added a sentence mentioning that this is provided by the starter at the top of this section.